Integrations/integration-KeyLight.yml

```diff
@@ -0,0 +1,418 @@
+description: Use LockPath KeyLight to manage tickets
+detaileddescription: For more information regarding thee integrations specific fields,
+  please advise the Cylance API Guide for version 1.7.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Port
+  name: port
+  defaultvalue: "4443"
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |
+    var server = params.server.replace(/[\/]+$/, '') + ':' + params.port + '/';
```

Comment:
Are you sure that regex works ?
I suspect it has some problems, but i might be wrong